pkgs/applications/virtualization/looking-glass-client/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, SDL2, SDL, SDL2_ttf, openssl, spice_protocol, fontconfig
+, libX11, freefont_ttf
+}:
+
+stdenv.mkDerivation rec {
+  name = "looking-glass-client-${version}";
+  version = "a10";
+
+  src = fetchFromGitHub {
+    owner = "gnif";
+    repo = "LookingGlass";
+    rev = version;
+    sha256 = "10jxnkrvskjzkg86iz3hnb5v91ykzx6pvcnpy1v4436g5f2d62wn";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ SDL SDL2 SDL2_ttf openssl spice_protocol fontconfig
```

Comment:
Actually, SDL does not seem to be used.

Reply:
If I don't include it, the build fails with:

```
make: *** [Makefile:29: .build/spice/spice.o] Error 1
In file included from main.c:24:0:
/nix/store/v4g1iqhh5y35ccc73bs82flijyvva2ys-SDL2_ttf-2.0.14/include/SDL2/SDL_ttf.h:34:17: fatal error: SDL.h: No such file or directory
 #include "SDL.h"
                 ^
compilation terminated.
make: *** [Makefile:29: .build/main.o] Error 1
```

Which seems a little weird (is the SDL2 library trying to find SDL1 headers or something?), but adding the SDL buildInput fixes it.

Reply:
Apparently, only these seem to be used:

```nix
$ nix-store --query --references (nix-build -A looking-glass-client)
/nix/store/z0b60y0khix9jb74ka56gw7b7n9s8awx-glibc-2.26-131
/nix/store/giigsd4cy3mb40v67albhiwc63s9djjj-freetype-2.7.1
/nix/store/2x3ia1b25g1wbmyks1llh9ns9qha7chw-fontconfig-2.12.1-lib
/nix/store/rb4vsahgqn5q4nq5k01vfqi6aiin4ax8-libX11-1.6.5
/nix/store/jp8i5dski55f02cn2xiyw0dpb3xn8xi0-mesa-noglu-17.2.8
/nix/store/gjdf0gwada330nyjmnzyrnz5vbhscn5a-glu-9.0.0
/nix/store/j705knck72ps4gpzk2jiswxj5nibxnqv-openssl-1.0.2n
/nix/store/pan0vwrvh40ciqfyjzrdjl9bxwbgc3jw-SDL2-2.0.7
/nix/store/v4g1iqhh5y35ccc73bs82flijyvva2ys-SDL2_ttf-2.0.14
```

Reply:
Hmm, you are right, those are needed by `SDL2`. We should probably propagate them from there.